big_scape/comparison/legacy_workflow_alt.py

```diff
@@ -162,8 +162,9 @@ def generate_edges(
                 if len(results) != len(task_batch):
                     raise ValueError("Mismatch between task length and result length")
 
-                for idx, pair in enumerate(task_batch):
-                    distance, jaccard, adjacency, dss = results[idx]
+                # for idx, pair in enumerate(task_batch):
+                #     distance, jaccard, adjacency, dss = results[idx]
+                for pair, distance, jaccard, adjacency, dss in results:
```

Comment:
In another branch I discovered this can be simplified:

```python
for a, b, c, d in some_tuple:
    yield (a, b, c, d)
```

is equivalent to

```python
yield from some_tuple
```